Automattic/jetpack
# Docs: Making symlinks and installation directory info more explicit

#### Changes proposed in this Pull Request:

* This PR makes information about symlinks and not cloning into the plugins directory a little more explicit.

#### Other information:


#### Jetpack product discussion

n/a

#### Does this pull request change what data or activity we track or use?

No.

#### Testing instructions:

* Proof-read. These edits are the result of a (private) conversation where it was realized that more explicit information on not cloning into the plugins directory would have saved some time. So testing here would really be following the links you would expect to find when trying to install / set up a local Jetpack environment for the first time (and you may not be using Docker).
